azure/services/managedclusters/spec_test.go

```diff
@@ -589,7 +589,6 @@ func getSampleManagedCluster() armcontainerservice.ManagedCluster {
 	return armcontainerservice.ManagedCluster{
 		Properties: &armcontainerservice.ManagedClusterProperties{
 			KubernetesVersion: ptr.To("v1.22.0"),
-			DNSPrefix:         ptr.To("test-managedcluster"),
```

Comment:
Was this deleted because `DNSPrefix` gets set to `s.DNSPrefix`?

Reply:
yes correct.